### ISSUE TITLE
November meetup

### ISSUE THREAD

**Issue by jakeret** [pydatazrh/meetup#33]
Let's try to schedule a meetup in november:

### Speakers (20min + 10min Q&A):
- awesome speaker A
- awesome speaker B
- awesome speaker C
- lightning talks

### Date
- xx.11.2017

### Location

- TDB

Have a room or something to talk about? Let us know!

**Comment by betatim** [comment]
Had a conversation today about meetup topics and conclusion was: tell me the story of analysing some data using python! What tools did you use, how did you decide to do what you did and what did you find out? Along the lines of I worked on problem X, I happened to use pydata tools. As opposed to here is a tool you could use.

Resulted in a tweet: https://twitter.com/betatim/status/919179286493450240 let's see what comes of it.

**Comment by jakeret** [comment]
Sounds nice! Would be cool if someone would engage on this

**Comment by betatim** [comment]
Emailed the opendata Zurich guys, let's see what happens.

**Comment by sredford** [comment]
Perhaps we don't need speakers to have a meetup? It would be nice to meetup regardless, there might still be some lightning talks and we could find out more about each other / what we do with python&data / what kind of talks we would like to hear.

**Comment by jakeret** [comment]
This is indeed true. @betatim and I are however considering to have a meetup early december. Wouldn't this be a bit to much, having two meetups so close to each other?

**Comment by betatim** [comment]
I like the idea. We could even use this to prepare a bit for December's blockchain special.

Some dates: 16 or 23 Nov for meetup and see what happens. Then December 12th or 14th for the Christmas special. For me that would be Ok/not too close.

**Comment by jakeret** [comment]
Ok let's do this. 16 nov and 12 dec works for me.
Wanna announce both?

**Comment by betatim** [comment]
Sent an email to ETH to ask about a room.

**Comment by TitusCornelius** [comment]
hey, I got a tip from a colleague that DigiComp, who are located close to
the train station, might be willing to host us for multiple future events
and provide snacks as well as drinks. I contacted them and will let you
know as soon as I hear back.

Do I understand that the 12.12. is our day of choice? Are there any other
options in case that this wouldn't work for them?

Am also working on some continuous financial support options for us. Will
keep you in the loop. I will be fully back regarding hands-on organization
of the meetups starting in February! Really looking forward to seeing you
all again!
